server/tables/tables.go

```diff
@@ -171,6 +171,8 @@ type Invocation struct {
 	RemoteExecutionEnabled bool
 
 	Tags string
+
+	ParentInvocationID string
```

Comment:
this field needs a non-unique index, otherwise the child invocations query will require a full table scan

Reply:
Ah good call - added